### DISCUSSION TITLE
Starting new conversations in python

### DISCUSSION THREAD

**Discussion by schipsy** [llm-workflow-engine/llm-workflow-engine#257] [Q&A] [answered]
When you provide too many messages in one chat (in terms of ChatGPT Web) I am running into a unresponsive answer after ~ 300 messages. I want to adapt the way i am giving it these messages by starting a new conversation after a certain point in a Python script. I could not find any documentation to do that? I tried some ways but the all (like direct cmd "ChatGPT /new" prompting) only results in the error that it would start a second session that is not working. 

Did I miss something in the documentation? I am using the Playwright tunnel

**Comment by thehunmonkgroup** [comment]
Not really enough data here to understand the issue.

You can try starting chatgpt with the `--debug` flag and check developer tools to see what the requests are doing.

Most likely you're just running into a limit imposed by the `chat.openai.com` website -- nothing we can do about that.

**Reply by schipsy** [reply]
thank you for the reply. You are right - but I am running into a specific solveable problem on the website. I just dont know how to use the "chatgpt /new" command in a phyton script. (something like bot.new is not working/available?)

**Comment by thehunmonkgroup** [comment]
check the `Backend` class, there's a method in there.

**Reply by schipsy** [reply]
Thank you for the answer, but I still can not get the "new_conversation" running. It is not starting a new conversation according to Web ChatGPT.

Here is my sample code (it is working but does not start the new conversation for the second part):

`from chatgpt_wrapper import ChatGPT
from chatgpt_wrapper.backends.browser.chatgpt import AsyncChatGPT



bot = ChatGPT()
controller = AsyncChatGPT()



success, response, message = bot.ask("Hello, world!")
if success:
    print(response)
else:
    raise RuntimeError(message)

controller.new_conversation()

success, response, message = bot.ask("Hello, to the new chat!")
if success:
    print(response)
else:
    raise RuntimeError(message)`

**Comment by thehunmonkgroup** [comment]
I'm not going to get into debugging code. You might consider hiring a developer to help you the rest of the way.

**Comment by schipsy** [comment, marked as answer]
Code update provided
